src/core/aws-utils.adb

```diff
@@ -883,34 +887,64 @@ package body AWS.Utils is
    -- Random --
    ------------
 
-   function Random return Random_Integer is
+   function Random (Task_Safe : Boolean := False) return Random_Integer is
```

Comment:
Shouldn't we be safe by default ? I mean in AWS most calls are with Task_Safe True now no?

Reply:
Changed to default True